lib/web/app/transport.go

```diff
@@ -156,6 +156,23 @@ func (t *transport) rewriteRequest(r *http.Request) error {
 	return nil
 }
 
+// websocketsDialer returns a function that dials a websocket connection
+// over the transport's reverse tunnel.
+func (t *transport) websocketsDialer() forward.Dialer {
+	return func(network, address string) (net.Conn, error) {
+		tr, ok := t.tr.(*http.Transport)
```

Comment:
hmm, why not save the TLS configuration in the transport so this is not necessary? Anything can trip over this if `t.tr` is ever an alternative implementation of `http.RoundTripper`

Reply:
Yeah, thought about it as well but for some reason decided to do without an additional field in transport. In hindsight, that'd probably be better, I'll update.